org/org.proto

```diff
@@ -180,6 +180,8 @@ message Organization {
   SuspensionStatus suspension_status = 19;
 
   string display_label = 20;
+
+  google.protobuf.Timestamp mfa_enforced_at =21;
```

Comment:
@sgagniere @channingdong your views on this?

Reply:
Also could you update the ticket title and description to be more detailed about why this change is needed and where it will be used? Instead of simply saying add `mfa_enforced_at`, say update organization object to include the new field `mfa_enforced_at` and in the description briefly describe where this change will be used in the cli repo and link the ticket in case in the future folks will make changes to this object as well and they want to know what happened before. This is a small ticket but it's a good habit to build long term.

Reply:
Let's match the Protobuf schema definition in cc-structs repo and put the `mfa_enforced_at` field as 23.